projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-grid.component.ts

```diff
@@ -184,6 +185,20 @@ export class IgxPivotGridComponent extends IgxGridBaseDirective implements OnIni
     @Output()
     public dimensionsChange = new EventEmitter<IDimensionsChange>();
 
+    /**
+     * Emitted when the dimension collection is changed via the grid chip area.
+     *
+     * @remarks
+     * Returns the new dimension collection and its type:
+     * @example
+     * ```html
+     * <igx-pivot-grid #grid [data]="localData" [height]="'305px'"
+     *              (dimensionsChange)="dimensionsChange($event)"></igx-grid>
+     * ```
+     */
+    @Output()
+    public pivotConfigurationChange = new EventEmitter<IPivotConfigurationChange>();
```

Comment:
Okay, so we're doing a `<prop>Change` pattern thing but not fully following it? If it was, this would have to emit the same type as `pivotConfiguration`. Was this intentional? Something messing with the two-way binding because of object refs?

Reply:
Yeah and since its a complex object two way data binding would be kinda weird and the users could get confused and need to set the whole object to a new instance while they will receive the same one. That's why I intentionally didn't follow the pattern so it is one way only but could easily allow for two way binding.

Reply:
In that case we might want to avoid using the patterns name if we're not following it, like `pivotConfigurationChanged` (with a `-d`). Still a bit unfortunate. 
Also if we're keeping the event args do name them as such -> IPivotConfigurationChangeEventArgs